common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -6084,6 +6091,18 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val SSL_HANDSHAKE_TIMEOUT_MS: ConfigEntry[Long] =
+    buildConf("celeborn.ssl.<module>.sslHandshakeTimeoutMs")
```

Comment:
Is `celeborn.ssl.<module>.sslHandshakeTimeoutMs` better to be internal? Because `celeborn.ssl.<module>.autoSslEnabled` is internal. BTW, the internal config option doesn't generate document.

Reply:
This would be relevant for any SSL connection - for example, between master/worker and spark application (unlike auto ssl, which is just within the application connections)

Reply:
Yes, this configuration is not going to be only used internally, but will also used for RPC communication from application to Celeborn Master/Workers.